admin/sql/CreateTables.sql

```diff
@@ -852,6 +852,24 @@ CREATE TABLE gender ( -- replicate
     gid                 uuid NOT NULL
 );
 
+CREATE TABLE genre ( -- replicate (verbose)
+    id                  SERIAL, -- PK
+    gid                 uuid NOT NULL,
+    name                VARCHAR NOT NULL,
+    comment             VARCHAR(255) NOT NULL DEFAULT ''
```

Comment:
Will genre aliases be managed through the edit system? (I assume so at some point, since the `genre_alias` table has an `edits_pending` column.) If so, we'll need an `edits_pending` column here too, since that's expected to exist by `Edit::Alias::Add` at the very least.

Reply:
Oh. I guess it should be the edit system? It's a bit weird in the sense that it will all probably be under the admin section at least for now, but if we hope to make them entities eventually (and I think we do) then having the *possibility* at least to do it via the edit system seems sensible.